sc_rf_service/sc_rf_service.py

```diff
@@ -236,7 +248,7 @@ class CavityPVGroup(PVGroup):
     rf_state_act: PvpropertyEnumRO = pvproperty(value=1, name='RFSTATE',
                                                 dtype=ChannelType.ENUM,
                                                 enum_strings=("Off", "On"),
-                                                read_only=True)
+                                                read_only=False)
```

Comment:
Oh, I changed it so that I could test the RF status bar on the front end side of our GUI. I think this was when I still had hope that I could add an outline around the status bar. I can change it back to read-only if that's preferred!

Reply:
That was me trying to actually replicate the machine (where we can't write to any ACTs lol). It's a philosophical question I suppose

Reply:
To clarify, I have the logic in place such that writing to RFCTRL writes to RFSTATE!